presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/ShardManager.java

```diff
@@ -30,7 +30,7 @@
     /**
      * Create a table.
      */
-    void createTable(long tableId, List<ColumnInfo> columns, boolean bucketed, OptionalLong temporalColumnId);
+    void createTable(long tableId, boolean tableSupportsDeltaDelete, List<ColumnInfo> columns, boolean bucketed, OptionalLong temporalColumnId);
```

Comment:
move `boolean tableSupportsDeltaDelete` after `boolean bucketed`

Reply:
> move `boolean tableSupportsDeltaDelete` after `boolean bucketed`

But from start, according to suggestion I move `boolean tableSupportsDeltaDelete` from back to the second position. Reason being we want this property closer to `tableId`